### PR TITLE
refactor: use word target instead of drive/device

### PR DESCRIPTION
**Pull request by Shou**
Change-Type: patch

### CODE REVIEW

File: lib/gui/app/pages/main/controllers/drive-selection.js [hunk 1]
```diff
@@ -39,10 +39,10 @@ module.exports = function (DriveSelectorService) {
 
     // eslint-disable-next-line no-magic-numbers
     if (drives.length === 1) {
-      return _.head(drives).description || 'Untitled Device'
+      return _.head(drives).description || 'Untitled Target'
```
Comment (jhermsmeier) [inline lib/gui/app/pages/main/controllers/drive-selection.js]: This would still be a device, imo

File: lib/shared/messages.js [hunk 2]
```diff
@@ -84,16 +84,16 @@ module.exports = {
...
 
     containsImage () {
-      return 'Drive Contains Image'
+      return 'Target Contains Image'
     },
 
     // The drive is large and therefore likely not a medium you want to write to.
     largeDrive () {
-      return 'Large Drive'
+      return 'Large Target'
```
Comment (jhermsmeier) [inline lib/shared/messages.js]: I'm thinking maybe this could be left as "Large Device", or maybe "Large Storage"?

File: lib/shared/messages.js [hunk 3]
```diff
@@ -156,8 +156,8 @@ module.exports = {
 
     notEnoughSpaceInDrive: () => {
       return [
-        'Not enough space on the drive.',
-        'Please insert larger one and try again.'
+        'Not enough space on the target.',
+        'Please insert a larger target and try again.'
```
Comment (jhermsmeier) [inline lib/shared/messages.js]: "insert a larger target" sounds weird to me – maybe we should use "insert a larger device" here, or rather "connect a larger device"?

File: lib/shared/messages.js [hunk 4]
```diff
@@ -200,17 +200,17 @@ module.exports = {
 
     driveUnplugged: () => {
       return [
-        'Looks like Etcher lost access to the drive.',
+        'Looks like Etcher lost access to the target.',
         'Did it get unplugged accidentally?',
-        '\n\nSometimes this error is caused by faulty readers that don\'t provide stable access to the drive.'
+        '\n\nSometimes this error is caused by faulty readers that don\'t provide stable access to the target.'
```
Comment (jhermsmeier) [inline lib/shared/messages.js]: Here, in the second mention, "target" seems awkwardly unspecific, maybe "target" -> "target device"?

File: lib/shared/messages.js [hunk 5]
```diff
@@ -200,17 +200,17 @@ module.exports = {
...
       ].join(' ')
     },
 
     inputOutput: () => {
       return [
-        'Looks like Etcher is not able to write to this location of the drive.',
-        'This error is usually caused by a faulty drive, reader, or port.',
-        '\n\nPlease try again with another drive, reader, or port.'
+        'Looks like Etcher is not able to write to this location of the target.',
+        'This error is usually caused by a faulty target, reader, or port.',
```
Comment (jhermsmeier) [inline lib/shared/messages.js]: "target" -> "device" here? Not sure a "faulty target" makes sense

File: lib/shared/messages.js [hunk 6]
```diff
@@ -200,17 +200,17 @@ module.exports = {
...
+        '\n\nPlease try again with another target, reader, or port.'
```
Comment (jhermsmeier) [inline lib/shared/messages.js]: Same here; target -> device?

File: lib/shared/messages.js (hunk 2 above)
Comment (lurch) [reply lib/shared/messages.js]: Yeah, "Large Target" makes me think of 
image

**Pull request closed without merging**